tests/test_helpers.ml

```diff
@@ -58,6 +58,24 @@ let _run_int command =
   | Unix.WEXITED v -> v
   | _ -> _command_failed command
 
+let _read_lines_from ~max_lines command =
```

Comment:
I think it's possible to use [`Sys.ocaml_version`](http://caml.inria.fr/pub/docs/manual-ocaml/libref/Sys.html#VALocaml_version) for this.